crates/aptos-rest-client/src/faucet.rs

```diff
@@ -51,6 +51,7 @@ impl FaucetClient {
         let response = self
             .inner
             .post(url)
+            .header("content-length", 0)
             .send()
```

Comment:
Hmm I thought I already did this by adding an empty body.  But maybe that was just the CLI.